usage problems

I would like to use lxml-stubs with my project and had difficulties to set it up. For debugging I constructed a simple testcase:
test.py 
```python
#!/usr/bin/python3
from lxml import etree

e = etree.fromstring("<ROOT/>")
print(etree.tostring(e))
```

Trying to lint this file failed - according to the given usage info I tried:
```bash
$> MYPYPATH=/my/path/to/lxml-stubs/ mypy test.py
test.py:2: error: Skipping analyzing 'lxml': found module but no type hints or library stubs
(...)
```
and
```bash
$> MYPYPATH=/my/path/to/lxml-stubs/lxml-stubs mypy test.py
test.py:2: error: Skipping analyzing 'lxml': found module but no type hints or library stubs
(...)
```

I solved this issue by renaming the inner lxml-stubs folder to lxml:
```bash
$> mv /my/path/to/lxml-stubs/lxml-stubs /my/path/to/lxml-stubs/lxml
$> MYPYPATH=/my/path/to/lxml-stubs/ mypy test.py
Success: no issues found in 1 source file
```

Is this the preferred way to use the stubs or am I missing something? If yes, I would suggest to rename the folder in the repo or update the usage info so that others will not have these problems.

Arno

--- a/README.md
+++ b/README.md
@@ -14,9 +14,18 @@ Eventually, it should become a PEP 561-style stubs package.
 
 ## Usage
 
-To use these stubs, clone this repo and point your type checker to it. For example, to use them in
-[mypy](http://github.com/python/mypy), you can set the `$MYPYPATH` environment variable or set
-`mypy_path = /path/to/lxml-stubs` in your `mypy.ini`.
+To use these stubs with [mypy](https://github.com/python/mypy), you have to
+install the `lxml-stubs` package.
+
+If you want to use the stubs as-is, you can build and install the package
+directly from its repository:
+
+    $ pip install https://github.com/lxml/lxml-stubs.git
+
+If you want to make local modifications, you can clone this repository and
+tell `pip` to install a link to the cloned source tree:
+
+    $ pip install -e /path/to/lxml-stubs
 
 ## Contributing
 

--- a/setup.py
+++ b/setup.py
@@ -1,12 +1,12 @@
-import glob
 from distutils.core import setup
 
 setup(
     name="lxml-stubs",
     version="0.1",
-    package_data={"lxml-stubs": glob.glob("lxml-stubs/*.pyi")},
+    package_data={"lxml-stubs": ["*.pyi"]},
     packages=["lxml-stubs"],
     classifiers=[
         "License :: OSI Approved :: Apache Software License",
     ],
+    zip_safe=False
 )

